test/testdrive/mzcompose.yml

```diff
@@ -8,29 +8,95 @@
 # by the Apache License, Version 2.0.
 
 version: '3.7'
+
+mzworkflows:
+  # Run tests with no support for s3
+  local:
+    steps:
+      - step: workflow
+        workflow: start-deps
+      - step: run
+        service: testdrive
```

Comment:
I think we shouldn't even support this as an mzcompose workflow. 200MB is nothing relative to the other stuff Docker puts you through.